tools/cldr-code/src/main/java/org/unicode/cldr/test/TestSupplementalData.java

```diff
@@ -53,43 +49,6 @@ public static void main(String[] args) throws IOException {
         checkTelephoneCodeData();
     }
 
-    private static void showMultiZones() {
-        // reverse the list
-        Relation<String, String> territoryToZones = Relation.<String, String> of(new TreeMap<String, Set<String>>(), TreeSet.class);
-        for (String zone : supplementalData.getCanonicalZones()) {
-            territoryToZones.put(supplementalData.getZone_territory(zone), zone);
-        }
-        // gather the data
-        // this could be slightly simpler using supplementalData.get
-        Set<String> singulars = new TreeSet<>();
-        for (String region : territoryToZones.keySet()) {
-            final Set<String> zones = territoryToZones.getAll(region);
-            if (zones.size() == 1 || region.equals("001")) {
-                singulars.addAll(zones);
-                continue;
-            }
-            System.out.println(region + "\t" + english.getName("territory", region));
-            System.out.println("\t" + zones);
-        }
-        XPathParts xpp = XPathParts.getFrozenInstance(root.getFullXPath("//ldml/dates/timeZoneNames/singleCountries"));
-        List<String> singleCountries = Arrays.asList(xpp.getAttributeValue(-1, "list").split("\\s+"));
```

Comment:
This crashed on line 75 since getAttributeValue returned null. Maybe this code hadn't been run in a long time.

The purpose of showMultiZones isn't clear. It called getExcludedZones and isPathExcludedForSurvey, which otherwise seem to be dead code. If there's a known benefit to showMultiZones, it needs fixing; otherwise deletion cleans up the code.